lib/cli.ts

```diff
@@ -232,7 +232,7 @@ function fix(argv: IArgv): Stream {
 
 function infer(argv: IArgv): Stream {
```

Comment:
@zbeekman a way to make it compile. Ugly though.

```suggestion
function infer(argv: any): Stream {
```

Reply:
@greut: That did not seem to resolve the problem (or, at least, it created a different one).

Reply:
the other _handlers_ needs to same treatment. Cheers mate and happy new year.